app/utils.ts

```diff
@@ -1,6 +1,8 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
```

Comment:
Why is this needed? I removed it locally and got no errors.

Reply:
nevermind, I see now it is the

```typescript
  const crypto = require("crypto");
```

in the ``generateRandomNumberString` function

Reply:
But maybe instead we should disable it for just that line instead of the entire file.